### DISCUSSION TITLE
While creating Pat Tokens from admin account  showing Beta version

### DISCUSSION THREAD

**Discussion by johit-cloudeq** [community/community#40432] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi

we are facing issue in creating pat token from admin account. while creating token, it   showing from  beta and same effecting on setting as well.

![MicrosoftTeams-image (4)](https://user-images.githubusercontent.com/83412449/204773533-0b08f6f4-d431-4ecf-816e-124bc5a7fac2.png)
![MicrosoftTeams-image (5)](https://user-images.githubusercontent.com/83412449/204773686-9c1c67c7-fe16-455a-9e95-08db9c81e7ee.png)

**Comment by ossanna16** [comment]
Hi there @johit-cloudeq and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
